Automattic/wp-calypso
# Hosting: Show upgrade banner to WoA sites on low-tier plans

Updates the hosting screen to work with any plan level for WP.com sites.

#### Changes proposed in this Pull Request

* Removes redirect for sites that don't fulfill the requirements of `can_view_hosting`.
* Updates feature check to SFTP, which is the hosting tools feature.

#### Testing instructions



* Navigate to /hosting-config and select any site.
* On WoA sites with a plan of Business and up it should show the hosting config.
* On Simple sites with a plan of Business and up it should show the activation banner.
* On WoA and Simple sites with lower-tier plan or no plan it should show an upgrade banner
* On Jetpack sites it should redirect to the site selector.



Related to p4TIVU-a66-p2
